generators/cloudfoundry/index.js

```diff
@@ -46,38 +46,38 @@ module.exports = CloudFoundryGenerator.extend({
             message: 'Name to deploy as:',
             default: this.baseName
         },
-        {
-            type: 'list',
-            name: 'cloudfoundryProfile',
-            message: 'Which profile would you like to use?',
-            choices: [
-                {
-                    value: 'dev',
-                    name: 'dev'
-                },
-                {
-                    value: 'prod',
-                    name: 'prod'
-                }
-            ],
-            default: 0
-        },
-        {
-            when: function(response) {
-                return databaseType != 'no';
+            {
```

Comment:
Hmm. Strange!
Honestly all I did was get IntelliJ Idea to format the code for me.
However I'll take another look.

I wonder why the build failed though - since I made no logical changes to the code.